AndroidSDK/src/com/leanplum/messagetemplates/MessageTemplates.java

```diff
@@ -91,17 +92,17 @@
     static final String DEFAULT_URL = "http://www.example.com";
 
     // Web interstitial values.
-    static final String DEFAULT_CLOSE_URL = "http://leanplum:close";
+    static final String DEFAULT_CLOSE_URL = "https://leanplum/close";
```

Comment:
Can you pull the base URL out into it's own value, so in the future we don't have to change multiple values?

```
DEFAULT_BASE_URL = "https://leanplum/"
static final String DEFAULT_CLOSE_URL = DEFAULT_BASE_URL + "close";
```